recipes/openssl/3.x.x/conanfile.py

```diff
@@ -631,6 +631,7 @@ def _module_file_rel_path(self):
     def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "OpenSSL")
         self.cpp_info.set_property("cmake_find_mode", "both")
+        self.cpp_info.set_property("cmake_target_name", "openssl::openssl")
```

Comment:
```suggestion
```

Unecessary and misleading since there is no such openssl::openssl in official FindOpenSSL.cmake

Reply:
Removed. thx!